pages/docs/getting-started/beta/quick-installation.md

```diff
@@ -10,40 +10,101 @@ Installing Open Data Hub requires OpenShift Container Platform version 4.8+. All
 
 ### Installing the Open Data Hub Operator
 
+You can either install the Open Data Hub Operator via Web GUI or via a subscription yaml. Decide that works best in your environment. Important to note is that we operate in namespace/project openshift-operators, whether installing the operator via GUI or subscription yaml.
+
+#### Installing the Open Data Hub Operator via Openshift Web GUI
 The Open Data Hub operator is available for deployment in the OpenShift OperatorHub as a Community Operators. You can install it from the OpenShift web console by following the steps below:
 
 1. From the OpenShift web console, log in as a user with `cluster-admin` privileges.  For a developer installation from [try.openshift.com](https://try.openshift.com/), the `kubeadmin` user will work.
 ![Log in to OpenShift]({{site.baseurl}}/assets/img/pages/docs/quick-installation/login.png "Log in to OpenShift")
-1. Create a new namespace named 'odh' for your installation of Open Data Hub.
+1. Create a new namespace, e.g. named 'odh' for your installation of Open Data Hub KfDefs later on.
 ![Create Namespace 'odh']({{site.baseurl}}/assets/img/pages/docs/quick-installation/create-namespace.png "Create Namespace 'odh'")
+1. Switch to the namespace 'openshift-operators' via navigation pane 'Home' - 'Projects'
```

Comment:
This explicit step is not required since the OLM UI will  default to a namespace with a `global-operators` operatorGroup which is `openshift-operators` by default